blueprints/ember-frost-fields/index.js

```diff
@@ -1,25 +1,29 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-hook', target: '~1.3.1'}
```

Comment:
ember hook in this file is ~1.3.1 but in package.json its ^1.4.1. They should probably match up.
i may not have noticed this in the other addons but just spotted it here.